src/selectors/roughHeights.js

```diff
@@ -183,6 +200,37 @@ function getScrollStateX(
   return ScrollbarState.HIDDEN;
 }
 
+/**
+ * @param {{
+ *   overflowX: string,
+ *   showScrollbarX: boolean,
+ * }} scrollFlags
+ * @param {number} height
+ * @param {number} scrollbarXHeight
+ * @param {number} scrollContentHeight
+ * @return {ScrollbarState}
+ */
+function getScrollStateY(
```

Comment:
For consistency, could you change `getScrollStateX` to take in `roughAvailableWidth` ?